style-guide/assessment-formats.md

```diff
@@ -0,0 +1,17 @@
+# Formatting Assessment Activities
+
+Assessment activities may take many formats, depending on the type of
+assessment.  Some assessment types include:
+
+* multiple choice questions
+* Jupyter notebook problems
```

Comment:
Does this describe an assessment "type" or "medium?"  I suspect I know what sorts of problems are conveniently distributed through notebooks, but I'm not sure the notebook defines the type of problem.